po/fr.po

```diff
@@ -1893,50 +1897,51 @@ msgstr "nom de l'attribut"
 
 #: rust/libnewsboat/src/filterparser.rs:64
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "L'un de: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
```

Comment:
This and the next message should start with a lowercase letter, because those translations are concatenated into a final error message that looks like this: `Parse error at position 42: expected one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#` or, with these translations: `Erreur d'analyse à la position42: L'un de: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !# était attentu`.

Also, I think there should be a space before the colon?